### PR TITLE
Added my video :Update video.yml

### PR DESCRIPTION
**Pull request by saarthakchats**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

File: _data/video.yml [hunk 1]
```diff
@@ -52,3 +52,8 @@
   title: Fossasia Summit
   img: FOSSASIA.jpeg
   author_name: Samyak Jain
+  
+ - url: https://www.youtube.com/watch?v=-EfJjFKBIYc
```
Comment (nikit19) [inline _data/video.yml]: Please remove the space before -

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please make this change

**Comment by saarthakchats** [comment]
@nikit19 Can we merge now?

**Review by realslimshanky — CHANGES_REQUESTED** [review]
Please squash your commits, use [this](https://davidwalsh.name/squash-commits-git) guide for help.

**Review by sayak119 — CHANGES_REQUESTED** [review]
Please resolve conflicts

**Comment by saarthakchats** [comment]
@sayak119 Resolved, pls merge

**Comment by saarthakchats** [comment]
@nikit19 Pls merge

**Comment by saarthakchats** [comment]
I have resolved all conflicts. But the big problem is that as soon as I squash, conflicts reappear. Then I have to add more commits. So can we please leave the squashing for this one last PR. As squashing creates conflicts. Pls merge and approve now.

**Review by harshit98 — APPROVED** [review]
Next time please squash your commits.

**Comment by saarthakchats** [comment]
@harshit98 Sure!

**Pull request merged**